plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -117,21 +117,6 @@
   {{ return(load_result('check_schema_exists').table) }}
 {% endmacro %}
 
-
-{% macro postgres__current_timestamp() -%}
-  now()
-{%- endmacro %}
-
-{% macro postgres__snapshot_string_as_time(timestamp) -%}
-    {%- set result = "'" ~ timestamp ~ "'::timestamp without time zone" -%}
-    {{ return(result) }}
-{%- endmacro %}
-
-
-{% macro postgres__snapshot_get_time() -%}
-  {{ current_timestamp() }}::timestamp without time zone
-{%- endmacro %}
-
 {#
   Postgres tables have a maximum length off 63 characters, anything longer is silently truncated.
```

Comment:
Not really your concern @colin-rogers-dbt 'cause your just innocently doing other work in this file, but maybe we can fix this small typo while you're already here?
```suggestion
  Postgres tables have a maximum length of 63 characters, anything longer is silently truncated.
```